### DISCUSSION TITLE
How to use GraphQL Apollo SSR?

### DISCUSSION THREAD

**Discussion by aspirisen** [shakacode/react_on_rails#1608] [Q&A]
Hello, is it possible to use [apollo ssr ](https://www.apollographql.com/docs/react/performance/server-side-rendering/) with react-on-rails? I have graphql requests in react components and want to ssr them.
Thanks.

**Comment by justin808** [comment]
Yes. If you get  https://www.shakacode.com/react-on-rails-pro/, it has examples.

**Comment by justin808** [comment]
@aspirisen  We're merging React on Rails Pro into this repo, so you can see examples soon.
